libr/core/cmd.c

```diff
@@ -3148,6 +3148,7 @@ R_API void r_core_cmd_init(RCore *core) {
 	r_cmd_add (core->rcmd, "info",     "get file info", &cmd_info);
 	r_cmd_add (core->rcmd, "cmp",      "compare memory", &cmd_cmp);
 	r_cmd_add (core->rcmd, "seek",     "seek to an offset", &cmd_seek);
```

Comment:
Yes, it should be and it is a certainly a step in the migration... Currently, can I add these spaghetti `cmd_*_init` mixed in the code so as to have no merge conflicts?